From 74e0c9bad60d4d616d1949daf140c75eebdb6483 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Fri, 10 Apr 2020 15:49:50 -0400
Subject: [PATCH 2/2] ze550kl: Green LED now defaults to on

This helps the user approximately know the moment the kernel is in
control of the hardware, compared to the bootloader.

Practically speaking, this is a sane default for Mobile NixOS and
non-android linux distros, as it shows the device is still on, as long
as there is no userspace LED management software changing the state.
---
 arch/arm/boot/dts/qcom/msm8939-cdp-ze550kl.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/msm8939-cdp-ze550kl.dtsi b/arch/arm/boot/dts/qcom/msm8939-cdp-ze550kl.dtsi
index 35e50b131dc12..c63b181d6daae 100644
--- a/arch/arm/boot/dts/qcom/msm8939-cdp-ze550kl.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8939-cdp-ze550kl.dtsi
@@ -372,8 +372,8 @@
 			qcom,led_mpp_4 {
 				label = "mpp";
 				linux,name = "green";
-				linux,default-trigger = "none";
-				qcom,default-state = "off";
+				linux,default-trigger = "default-on";
+				qcom,default-state = "on";
 				qcom,max-current = <40>;
 				qcom,current-setting = <5>;
 				qcom,id = <6>;
-- 
2.23.1

